lib/IDE/CodeCompletionResultType.cpp

```diff
@@ -234,21 +245,33 @@ const USRBasedType *USRBasedType::fromType(Type Ty, USRBasedTypeArena &Arena) {
       auto Conformances = Nominal->getAllConformances();
       Supertypes.reserve(Conformances.size());
       for (auto Conformance : Conformances) {
+        if (isa<InheritedProtocolConformance>(Conformance)) {
+          // Skip inherited conformances; we'll collect them when we visit the
+          // superclass.
+          continue;
+        }
         if (Conformance->getDeclContext()->getParentModule() !=
             Nominal->getModuleContext()) {
           // Only include conformances that are declared within the module of the
           // type to avoid caching retroactive conformances which might not
           // exist when using the code completion cache from a different module.
           continue;
         }
-        if (Conformance->getProtocol()->isSpecificProtocol(KnownProtocolKind::Sendable)) {
+        if (Conformance->getProtocol()->isSpecificProtocol(KnownProtocolKind::Sendable) ||
+            Conformance->getProtocol()->getInvertibleProtocolKind()) {
           // FIXME: Sendable conformances are lazily synthesized as they are
           // needed by the compiler. Depending on whether we checked whether a
           // type conforms to Sendable before constructing the USRBasedType, we
           // get different results for its conformance. For now, always drop the
           // Sendable conformance.
+          //
+          // FIXME: Copyable and Escapable conformances are skipped because the
+          // USR mangling produces the type 'Any' for the protocol type.
```

Comment:
Why does it produce `Any` for the protocol name?

Reply:
Because we need to fix the mangling.

Reply:
Ah, I see. Thanks for the clarification. Is the name mangling something we are planning to fix in the short term?

Reply:
OK, great 👍🏽